<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myWidget</class>
 <widget class="QWidget" name="myWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Slider和QColor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Red</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="SliderRed">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Green</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="SLiderGreen">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Blue</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="SliderBlue">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Alpha</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSlider" name="SliderAlpha">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Dial和LCDNUmber</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QDial" name="dial">
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="notchesVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="LCDDispaly"/>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>LCD显示进制</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>30</number>
         </property>
         <item>
          <widget class="QRadioButton" name="rbtDec">
           <property name="text">
            <string>十进制</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtBin">
           <property name="text">
            <string>二进制</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtOct">
           <property name="text">
            <string>八进制</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtHex">
           <property name="text">
            <string>十六进制</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
